book/tables/adverse-events/aet01_aesi.qmd

```diff
@@ -481,6 +483,8 @@ result <- build_table(lyt_adae, df = adae_smq, alt_counts_df = adsl)
 result
 ```
 
+{{< include ../../test-utils/save_results.qmd >}}
+
 ## Teal
 
 ```{r teal, message=FALSE}
```

Comment:
```suggestion
```{r teal, message=FALSE, opts.label='skip_if_testing'}
```
I think it makes sense to skip them all

Reply:
Agreed, Thanks for making the change in this PR. I will make sure they are there in other PRs as well.